executor/src/witgen/processor.rs

```diff
@@ -52,34 +53,28 @@ impl<'a, T: FieldElement> SolverState<'a, T> {
 #[derive(Clone)]
 pub struct OuterQuery<'a, 'b, T: FieldElement> {
     /// Rows of the calling machine.
-    pub caller_rows: &'b RowPair<'b, 'a, T>,
+    pub range_constraints: &'b dyn RangeConstraintSet<AlgebraicVariable<'a>, T>,
     /// Connection.
     pub connection: Connection<'a, T>,
     /// The left side of the connection, evaluated.
-    pub left: Left<'a, T>,
+    pub parameters: Left<'a, T>,
```

Comment:
```suggestion
    pub arguments: Left<'a, T>,
```

or just `inputs`?

Reply:
I'll stick with arguments then, for consistency